tools/harness-thci/OpenThread.py

```diff
@@ -66,6 +66,11 @@
 import commissioner
 from commissioner_impl import OTCommissioner
 
+# Replace by the actual version string for the vendor's reference device
+OT11_VERSION = 'ot11_version'
```

Comment:
```suggestion
OT11_VERSION = 'OPENTHREAD'
```
I think the default configuration is too strict. 
We may still use `OPENTHREAD` as default?